app/components/primer/button_group_component.rb

```diff
@@ -5,6 +5,9 @@ module Primer
   class ButtonGroupComponent < Primer::Component
     include ViewComponent::SlotableV2
 
+    # Required list of buttons to be rendered.
+    #
+    # @param kwargs [Hash] the same arguments as <%= link_to_component(Primer::ButtonComponent) %>
```

Comment:
What do you think having this import all of the arguments from `ButtonComponent`, so we don't have to link to it?

Reply:
we can do it as a next step! I don't want this PR to become overcomplicated